pkgs/by-name/na/nautilus/wrapper.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  makeWrapper,
+  symlinkJoin,
+  nautilus,
+  nautilusExtensions,
+}:
+
+symlinkJoin {
+  name = "nautilus-with-extensions-${nautilus.version}";
+
+  paths = [ nautilus ] ++ nautilusExtensions;
```

Comment:
We probably ought to check that all paths contain extension directory. Unfortunately, will have to be done at build time, if we do not want to go the way of adding `passthru.nautilusExtension` to every extension package and check that.